Aztec/Classes/NSAttributedString/VisualOnlyElementFactory.swift

```diff
@@ -31,7 +31,7 @@ class VisualOnlyElementFactory {
     func zeroWidthSpace(inheritingAttributes inheritedAttributes: [String:Any]? = nil) -> NSAttributedString {
         var attributes = inheritedAttributes ?? [String:Any]()
 
-        attributes[VisualOnlyAttributeName] = VisualOnlyElement.zeroWidthSpace.rawValue
+        //attributes[VisualOnlyAttributeName] = VisualOnlyElement.zeroWidthSpace.rawValue
```

Comment:
This fixes the issue, but of course it's not ideal, that is why I wanted some guidance on the visual to Dom mappings.